packages/opal-common/opal_common/git/branch_tracker.py

```diff
@@ -26,6 +27,7 @@ def __init__(
         branch_name: str = "master",
         remote_name: str = "origin",
         retry_config=None,
+        ssh_key: Optional[str] = None,
```

Comment:
please add it to the args as well, and if we support only one type of keys it would be good to mention (The best you can do in my opinion is to explain how to create such key or link to such docs)